### PR TITLE
[16.0][MIG] sales_team_security

### PR DESCRIPTION
**Pull request by SoporteOperu**

### CODE REVIEW

File: sales_team_security/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+#: model:ir.model.fields,help:sales_team_security.field_res_users__user_id
+msgid "The internal user in charge of this contact."
+msgstr "El usuario interno a cargo de este contacto."
+
+#. module: sales_team_security
+#: model:res.groups,comment:sales_team_security.group_sale_team_manager
+msgid ""
+"The user will have an access to the documents of the sales teams he/she "
+"belongs to."
+msgstr ""
```
Comment (Yadier-Tecnativa) [inline sales_team_security/i18n/es.po:61]: Translation suggested:
El usuario tendrá acceso a los documentos de los equipos comerciales a los que pertenece.

File: sales_team_security/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+#. module: sales_team_security
+#: model:res.groups,name:sales_team_security.group_sale_team_manager
+msgid "User: Team Documents Only"
+msgstr ""
```
Comment (Yadier-Tecnativa) [inline sales_team_security/i18n/es.po:66]: Translation suggested:
Usuario: Solo documentos del equipo

**Review by Yadier-Tecnativa — COMMENTED** [review]
Tested in runboat, works fine!
Approved

**Comment by Yadier-Tecnativa** [comment]
ping @pedrobaeza

**Review by Yadier-Tecnativa — APPROVED** [review]
good

**Comment by Yadier-Tecnativa** [comment]
Supersede by https://github.com/OCA/sale-workflow/pull/2602

**Comment by pedrobaeza** [comment]
Check my comment https://github.com/OCA/sale-workflow/pull/2602#issuecomment-1633591684

**Comment by rousseldenis** [comment]
@SoporteOperu Do you mind if we switch to #2602 ?

**Comment by rafaelbn** [comment]
Closing as we don't have answer of @SoporteOperu in 2 weeks

**Pull request closed without merging**